webpack.plugins.config.js

```diff
@@ -0,0 +1,53 @@
+var webpack = require('webpack');
+var _       = require('underscore');
+
+var plugins = {
+  envPlugin: function(type) {
+    // Sepecify which environment variables
+    return new webpack.DefinePlugin({
+      'process.env': {
```

Comment:
Related to the other comment - let's just use `DEBUG` instead.

This is actually a bit strange - the expectation from the other line is that we're using the [EnvironmentPlugin](https://webpack.js.org/plugins/environment-plugin/), which is a shortcut for the DefinePlugin and env variables. In that case I'd expect that there'd be changes to our build commands to set different env variables when running different stages, and the question would be if we wanted to use node variables in client side code. But, here we're not doing that, and also having to drop into (or explicitly define) process.env variables.